static/app/views/detectors/routes.tsx

```diff
@@ -17,3 +17,8 @@ export const detectorRoutes = (
     </Route>
   </Route>
 );
+
+export const MONITOR_ROUTES = {
+  LIST: () => '/issues/monitors/',
+  DETAILS: (monitorId: string) => `/issues/monitors/${monitorId}/`,
+};
```

Comment:
Nit we could simplify to:
```suggestion
export const MONITOR_RULES_BASE_URL = '/issues/monitors'
```
Then we can just use like: `${MONITOR_RULES_BASE_URL}/{id}` in code. The goal I'm hoping for is to DRY up the `'/issues/monitors/` string.

Reply:
fixed in https://github.com/getsentry/sentry/pull/90004/commits/ef9c7c4e5fd20406d6e0ef0fecb26de213a1fc0e